src/models/Illness.ts

```diff
@@ -0,0 +1,15 @@
+
+import { IllPerson } from './fields/IllPerson';
+import { YesNo } from './fields/YesNo';
+
+import { Attachment } from 'app/models/Attachment';
+
+export interface Illness {
+    illPerson: IllPerson;
+    otherPerson?: string;
+    illnessStart: Date;
+    continuedIllness: YesNo;
```

Comment:
I'd prefer to have only properties which are needed for delivery of the story. Rest could be shared with the team and put under discussion if not done yet.

Reply:
Noted and ammended.